public/app/plugins/panel/debug/EventBusLogger.tsx

```diff
@@ -52,7 +52,7 @@ export class EventBusLoggerPanel extends PureComponent<Props, State> {
 
   eventObserver: PartialObserver<BusEvent> = {
     next: (event: BusEvent) => {
-      const origin = event.origin as any;
+      const origin: any = event.origin;
```

Comment:
lol i guess

Reply:
i _know_, but number still needs to go down before i can face actually tackling problems 😂